proto/api/v1/project.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+package api.v1;
+
+import "api/v1/common.proto";
+import "google/protobuf/timestamp.proto";
+import "validate/validate.proto";
+
+// ProjectService serves project related functions
+service ProjectService {
+  // List all accessible projects
+  rpc List(ProjectServiceListRequest) returns (ProjectServiceListResponse) {
+    option (visibility) = VISIBILITY_SELF;
+    option (auditing) = AUDITING_EXCLUDED;
+  }
+  // Get a project
+  rpc Get(ProjectServiceGetRequest) returns (ProjectServiceGetResponse) {
+    option (project_roles) = PROJECT_ROLE_OWNER;
+    option (project_roles) = PROJECT_ROLE_EDITOR;
+    option (project_roles) = PROJECT_ROLE_VIEWER;
+    option (auditing) = AUDITING_EXCLUDED;
+  }
+}
+
+// Project defines a group of resources belonging to a tenant
+// a tenant can have multiple projects
+message Project {
+  // Uuid of this project
+  string uuid = 1;
+  // Name of this project
+  string name = 2;
+  // Description of this project
+  string description = 3;
+  // Tenant this project belongs to
+  string tenant = 4;
+  // CreatedAt the date when this project was created
+  google.protobuf.Timestamp created_at = 10;
```

Comment:
Is numbering starting at 10 for timestamps intended?

Reply:
yes to leave room for more properties, created and updated times are always kept at the end